region/brinstar/kraid/Kraid Room.json

```
@@ -256,7 +256,7 @@
     {
       "id": 36,
       "link": [1, 1],
-      "name": "X-Mode + Crystal Flash (Kraid Alive) - R-Mode Spark Interrupt (Gain Blue Suit)",
+      "name": "Kraid Phase 1, Crystal Flash, X-Mode Shinecharge, R-Mode Spark Interrupt",
```

I would assume this can be done on P2 as well? It's a very long strat name, so shortening it can help
```suggestion
      "name": "Kraid, Crystal Flash, X-Mode Shinecharge, R-Mode Spark Interrupt",
```

---

> Maybe: `Early Crystal Flash, X-Mode ...` ?

I would just remove Early. Should be clear by the note. It's good to not have too long of strat names for the spoiler log.  I am happy to merge this regardless of what you decide to name it though, just lmk